/**:
  ros__parameters:
    car_width: 0.27
    car_length: 0.5
    front_overhang: 0.43
    rear_overhang: 0.08
    wheel_base: 0.324
    steering_speed: 3.2
    scan_down_sample_scale: 2.0
    range_thresh_min: 3.0
    range_thresh_max: 6.0
    min_gap_size: 0.8
    goal_angle: 0.0
    wall_clearance_min: 0.26
    wall_clearance_max: 0.5
    wall_clearance_dist_thresh: 1.8
    min_speed: 1.1
    slow_speed: 3.1
    max_speed: 5.5
    boost_speed: 5.8
    max_speed_angle_thresh: 4.9
    boost_dist_thresh: 4.6
    acceleration: 4.2
    deceleration: 12.0
    small_angle_kp: 0.5
    large_angle_kp: 0.65